src/page/template/detail-view.htm

```diff
@@ -6,29 +6,31 @@
           <div class="label-bold-xs" data-bind="text: messageEntity().unsafeSenderName(), attr: {'data-uie-uid': messageEntity().user().id, 'data-uie-value': messageEntity().user().name}" data-uie-name="fullscreen-picture-sender"></div>
           <div class="label-xs" data-bind="text: formatTimestamp(messageEntity().timestamp()), attr: {'data-timestamp': messageEntity().timestamp()}"></div>
         </div>
-        <close-icon class="detail-view-header-close-button icon-button" data-bind="click: clickOnClose" data-uie-name="do-close-detail-view"></close-icon>
+        <button type="button" class="detail-view-header-close-button icon-button" data-bind="click: clickOnClose, attr: {'aria-label': t('conversationDetailsCloseLabel')}" data-uie-name="do-close-detail-view">
+          <close-icon></close-icon>
+        </button>
       </header>
-      <div class="detail-view-main" data-bind="click: clickOnClose">
-        <img class="detail-view-image" data-bind="attr: {src: imageSrc()}" data-uie-name="status-picture"/>
+      <div class="detail-view-main" data-bind="click: clickOnClose, event: { keydown: (_, event) => hadndlePressOnClose(event) }, attr: {'aria-label': t('conversationDetailsCloseLabel')}" tabindex="0" role="button">
+        <img class="detail-view-image" data-bind="attr: {src: imageSrc()}" data-uie-name="status-picture" alt=""/>
```

Comment:
maybe the alt should share the data-uie-name value?

Reply:
Empty alt is ok here, as this image is only a presentation one. Additionally, we have a "button" with a label on it above so the reader will read this label :)